bin/benchmark_elbo.jl

```diff
@@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat")
 srand(1)
 println("Loading data.")
 
-S = 100
+S = 3
```

Comment:
Yep.  (I do know how to use the profiler, so what would be helpful is
another pair of eyes looking for ideas to cut out steps.)

On Thu, Jan 21, 2016 at 4:34 PM, Jeffrey Regier <notifications@github.com>
wrote:

> In bin/benchmark_elbo.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/103#discussion_r50487990>:
>
> > @@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat")
> >  srand(1)
> >  println("Loading data.")
> >
> > -S = 100
> > +S = 3
>
> Looks like `combine_sfs!` alone is taking 78 seconds, whereas the whole
> elbo with derivs used to take 24 seconds, so that's where I'd start. ```
> 78032 ...e/src/SensitiveFloat.jl combine_sfs! 138 ```
> … <#-1952332602_>
> On Thu, Jan 21, 2016 at 2:27 PM Ryan <notifications@github.com> wrote: In
> bin/benchmark_elbo.jl <#103 (comment)
> <https://github.com/jeff-regier/Celeste.jl/pull/103#discussion_r50474863>>:
> > @@ -12,7 +12,7 @@ const dat_dir = joinpath(Pkg.dir("Celeste"), "dat") >
> srand(1) > println("Loading data.") > > -S = 100 > +S = 3 Pushed. With lots
> of sources it's actually worse than with a single source -- about 5x
> slower. PTAL and let me know what you see. — Reply to this email directly
> or view it on GitHub <
> https://github.com/jeff-regier/Celeste.jl/pull/103/files#r50474863>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/103/files#r50487990>.
>